### PR TITLE
[exporter/doris] Mark the Doris exporter as Alpha

### PR DESCRIPTION
**Pull request by joker-star-l**
#### Description
Mark the Doris exporter as Alpha

#### Link to tracking issue
#33479

### CODE REVIEW

File: cmd/otelcontribcol/builder-config.yaml [hunk 1]
```diff
@@ -504,3 +505,4 @@ replaces:
   - github.com/open-telemetry/opentelemetry-collector-contrib/pkg/status => ../../pkg/status
   - github.com/open-telemetry/opentelemetry-collector-contrib/receiver/awss3receiver => ../../receiver/awss3receiver
   - github.com/DataDog/datadog-api-client-go/v2 => github.com/DataDog/datadog-api-client-go/v2 v2.31.0
+  - github.com/open-telemetry/opentelemetry-collector-contrib/exporter/dorisexporter => ../../exporter/dorisexporter
```
Comment (bogdandrutu) [inline cmd/otelcontribcol/builder-config.yaml:508]: new line please.

**Review by bogdandrutu — APPROVED** [review]

File: cmd/otelcontribcol/builder-config.yaml (hunk 1 above)
Comment (joker-star-l) [reply cmd/otelcontribcol/builder-config.yaml:508]: Do I need to add a blank line?

**Comment by joker-star-l** [comment]
Hello @atoulme , could you help me review this PR?

**Review by atoulme — APPROVED** [review]

File: cmd/otelcontribcol/builder-config.yaml (hunk 1 above)
Comment (atoulme) [reply cmd/otelcontribcol/builder-config.yaml:508]: yes

**Review by evan-bradley — APPROVED** [review]
Thanks @joker-star-l. Please open a pr in opentelemetry-collector-releases to add it to the contrib distro [here](https://github.com/open-telemetry/opentelemetry-collector-releases/blob/main/distributions/otelcol-contrib/manifest.yaml) when you are able.

**Pull request merged**